src/execute/oas3/build-request.js

```diff
@@ -147,14 +147,15 @@ export function applySecurities({request, securities = {}, operation = {}, spec}
           }
         }
         else if (type === 'http') {
-          if (schema.scheme === 'basic') {
+          const scheme = schema.scheme && schema.scheme.toLowerCase()
```

Comment:
We should be able to use lodash [toLower](https://lodash.com/docs/4.17.15#toLower) which returns and empty string of when provided a non-string argument.

```js
import 'toLower' from 'lodash/toLower';
```
```suggestion
          const scheme = toLower(schema.scheme)
```